### PR TITLE
Populate post from yaml

### PR DESCRIPTION
**Pull request by thvu11**
`./bin/post-to-ghost.sh` will handle following content:
- html content of posts
- link to PDF files
- link to images
- publication history
- tags

A sample login detail is set as:
- Username: test01@mail.com
- Password: long_pass_01

### CODE REVIEW

File: data/backup.yml [hunk 1]
```diff
@@ -66040,7 +66040,7 @@ attributes:
       \r\n\
       Or watch a longer excerpt of Leslie's talk on ABC News 24. Stay tuned for specific dates and times when the program will go to air. "
     link: ""
-    publish_on: April 01, 2011
```
Comment (adamc00) [inline data/backup.yml]: Altering the input file to match your input requirements should be avoided. It's better to either process the data on the fly into the format you want or write out a new file for input into the next stage. While the file is in version control so you can always go back and see what you have done it is better to keep the supplied source around so you can always easily see what the original content looked like.

**Review by adamc00 — CHANGES_REQUESTED** [review]

**Comment by thvu11** [comment]
I revert that commit and add a format time check for that case

**Comment by thvu11** [comment]
You have issue running the export?

**Comment by adamc00** [comment]
> You have issue running the export?

Just accidental commit of logs from ghost. I've added a .gitignore for them.

**Comment by thvu11** [comment]
Tags, publication history and content link has been added.

**Review by adamc00 — APPROVED** [review]

**Pull request merged**